pkg/util/fsnotification/doc.go

```diff
@@ -0,0 +1,3 @@
+// Package fsnotification provides helper functions that wrapper the fsnotify filesystem
```

Comment:
Isn't "wrapper" a noun? http://www.dictionary.com/browse/wrapper
And the "to wrap" verb goes "it wraps" and "they wrap"?
Really *they wrapper*? Looks weird to me.